pkg/services/playlist/playlistimpl/object_store.go

```diff
@@ -73,8 +75,10 @@ func (s *objectStoreImpl) Create(ctx context.Context, cmd *playlist.CreatePlayli
 			return rsp, fmt.Errorf("unable to write playlist to store")
 		}
 		_, err = s.objectstore.Write(ctx, &object.WriteObjectRequest{
-			UID:  rsp.UID,
-			Kind: models.StandardKindPlaylist,
+			GRN: &object.GRN{
+				UID:  rsp.UID,
```

Comment:
this and many other places are missing the orgId - I assume thats the next step? :)

Reply:
for query, orgId does not *need* to be specified since it can be pulled of the user -- if it is specified, we need to verify that userId is the same as the orgId 

:)